src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.scss

```diff
@@ -236,6 +236,23 @@
     }
   }
 }
+
+//detail panel
+.detail-panel {
+  background: $color-pf-white;
+  position: fixed;
+  left: auto;
+  top: em(34);
+  right: 0;
+  bottom: 0;
+  width: 50%;
```

Comment:
This is the width of the panel, i.e. half of the page. How to convert % into em? @SMahil any idea?